addons/crm/tests/test_crm_ui.py

```diff
@@ -8,3 +8,17 @@ class TestUi(odoo.tests.HttpCase):
 
     def test_01_crm_tour(self):
         self.start_tour("/web", 'crm_tour', login="admin")
+
+    def test_02_crm_tour_rainbowman(self):
+        # we create a new user to make sure he gets the 'Congrats on your first deal!'
+        # rainbowman message.
+        self.temp_crm_user = self.env['res.users'].create({
```

Comment:
is it really useful to set the user on `self`?

Reply:
It is not, left over from my copy/paste.
Thanks!